docs/package_docs.rst

```diff
@@ -86,52 +86,101 @@ We suggest this template:
 
    See LICENSE and COPYRIGHT files.
 
-.. _pkg-doc-license:
-
-LICENSE
--------
-
-A ``LICENSE`` file in the repository's root is the canonical description of LSST's code licensing.
-
-The canonical source of this file's content is https://www.lsstcorp.org/LegalNotices/LsstLicenseStatement.txt, though ensure that it is named ``LICENSE`` in the repository.
-
 .. _pkg-doc-copyright:
 
 COPYRIGHT
 ---------
 
-We assert copyright information in a centralized ``COPYRIGHT`` file, located in the repository's root.
+We assert copyright information in a centralized ``COPYRIGHT`` file, located in the repository's root rather than attempting to track copyright in each file separately within a package.
```

Comment:
Suggested edit:

```
We assert copyright information in a centralized ``COPYRIGHT`` file, located in the repository's root, instead of in individual source files.
```

The next sentence does the job of the "attempting to..." sentiment I think.

Reply:
You could even slim down the next sentence to just

```
See `RFC-45 <https://jira.lsstcorp.org/browse/RFC-45>`_ for background on this policy.
```

Reply:
@ktlim has previously expressed a desire not to use RFCs as part of the documentation if we can avoid it. I tend to agree that RFC-45 is not a coherent view of policy. What it shows is a lot of angst.

Reply:
Sounds good. I'd just avoid the word "attempting" since it sets the wrong tone.